### PR TITLE
[website] "hello world" blog entry

### LINKED ISSUE #464: [website] ClusterLink introduction blog

### PR DESCRIPTION
**Pull request by elevran**
Still in draft mode.
Tried to write an intro blog in the "first person" - please review for tone as well, not sure it if that's what we want.
I will add a more technical/serious entry as well.

Fixes #464

### CODE REVIEW

**Review by michalmalka — APPROVED** [review]
I liked it. As this is still in draft I didn't fixes typos or such... will do it when the blog will be ready.

**Comment by elevran** [comment]
It is draft since I did not want it published without approval due to tone. We can remove the draft and put future date in it.
Hugo ha two flags - `buildFuture` and `buildDrafts` which control publishing based on draft status and current date.

File: website/content/en/blog/hello-world.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+author: Etai Lev Ran
+description: >-
+     Welcome to ClusterLink
+type: blog
+draft: true
+---
+
+Ah, hello - anyone there?
+
+I'm new here and wanted to say hi and formally introduce myself. Hi - I'm ClusterLink, and
```
Comment (michalmalka) [inline website/content/en/blog/hello-world.md:14]: I think you should go newline before the Hi...

File: website/content/en/blog/hello-world.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ I'm the new kid on the block - just starting out as open-source project trying to
+ find my place in the big, wide world of multicluster Kubernetes connectivity. This is my
+ first venture away from my [home on GitHub](https://github.com/clusterlink-net/clusterlink)
+
+At my core, I offer a simple yet powerful philosophy which I like to think of the three S'es:
+ **Seamless**, **Simple**, and **Secure** Service Interconnect. Maybe it should have been the
+ four S'es? Those off by one errors were always pesky... Anyway, wWhere was I? Ah - yes:
```
Comment (michalmalka) [inline website/content/en/blog/hello-world.md:21]: Typo... where (unnecessary W)

**Review by michalmalka — APPROVED** [review]

**Pull request merged**